ext/mysqli/mysqli_api.c

```diff
@@ -2254,7 +2254,7 @@ PHP_FUNCTION(mysqli_ssl_set)
 		}
 	}
 
-	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4]);
+	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4], NULL);
```

Comment:
I think this will break the build against libmysql. `mysql_ssl_set` is a mysql API emulated by mysqlnd, so I don't think it's possible to simply change the signature.